src/jit/gentree.h

```diff
@@ -2671,7 +2671,7 @@ struct GenTreeCall final : public GenTree
 
 #ifdef FEATURE_READYTORUN_COMPILER
     // Call target lookup info for method call from a Ready To Run module
-    CORINFO_CONST_LOOKUP gtEntryPoint;
+    CORINFO_LOOKUP gtEntryPoint;
```

Comment:
`CORINFO_LOOKUP` is very large. This will impact JIT memory consumption and throughput. It needs to be done differently.